frontend/src/app/shared/components/account/account.component.ts

```diff
@@ -129,6 +132,8 @@ export class AccountComponent implements OnInit {
               this.accountModalService.openWelcomeDialog();
               this.cartService.sendNewCartItems();
               this.cartService.getCartItems();
+              this.favouritesService.sendNewFavouritesItems();
+              this.favouritesService.getFavouritesItems();
```

Comment:
2nd request may be performed before 1st. There is no any guarantee which will be first. Should be united, because there is no any need to perform two consequent (parallel is absolutely crime) requests

Reply:
very important!